src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/BDDRangeConverter.cs

```diff
@@ -22,26 +22,26 @@ internal sealed class BDDRangeConverter
         /// Convert the set into an equivalent array of ranges.
         /// The ranges are nonoverlapping and ordered.
         /// </summary>
-        public static (uint, uint)[] ToRanges(BDD set, int maxBit)
+        public static (uint, uint)[] ToRanges(BDD set)
         {
-            Debug.Assert(0 <= maxBit && maxBit <= 31, "maxBit must be between 0 and 31");
+            const int MaxBit = 15;
```

Comment:
NIT: Should we add a comment here why 15 was chosen?

Reply:
Done